### ISSUE TITLE
Unable to install on Sublime Text 3

### ISSUE THREAD

**Issue by imcatnoone** [imcatnoone/toothpaste#2]
Folks receive the following error `Error loading colour scheme Packages/toothpaste.tmTheme: Error parsing plist xml: expected = in file Packages/toothpaste.tmTheme on line: 50`

**Comment by tomchewitt** [comment]
I managed to manually install in sublime text 3 by cloning the repo (rather than saving the file). Googling the XML error led here: https://github.com/blacked/zbx_nginx_template/issues/4

After cloning I then just copied and pasted the theme file into the destination in the README. All works great.

Hope this helps.

**Comment by imcatnoone** [comment]
@tomchewitt thanks so much for this feedback. was super helpful! adding this tidbit to the readme. Mind if I add you to the contributors list on the README? Also, can you give the README a check and see if those installation instructions are correct with the approach you took? Thanks!

**Comment by tomchewitt** [comment]
@imcatnoone All looks good to me, happy to be on the contribution list :)

**Comment by skreutzberger** [comment]
@tomchewitt Can you please explain in detail how you got it work? I did exactly as described in the README for the latest Sublime Text 3 and the theme still does not show up under Preferences 🤔

**Comment by imcatnoone** [comment]
Hey @skreutzberger have you checked the latest instructions recently—as in, the last week?

**Comment by skreutzberger** [comment]
yes, I checked them today :) 

I actually found out the issue by myself some minutes ago. To correctly install the `.thTheme` file under Sublime Text 3 you need to install the Sublime package `Colorsublime` first. And then you need to follow [these instructions](http://colorsublime.com/how-to-install-a-theme) and copy `toothpaste.tmTheme` to the newly created `Colorsublime-Themes` folder.

**Comment by imcatnoone** [comment]
@skreutzberger slightly delayed here but thanks so much for this!

**Comment by DeeprajPandey** [comment]
Yeah, exactly what @skreutzberger said but the link to the instructions is broken and we don't have to make a Colorsublime folder for that matter.

I'm just listing out the steps so it comes in handy for others.
Go to
`~/library/Application Support/Sublime Text 3/packages`

Create a new folder "Toothpaste"
`mkdir Toothpaste`

Copy the `toothpaste.tmTheme` file ([here](https://github.com/imcatnoone/toothpaste/blob/master/toothpaste.tmTheme)) to the `Toothpaste` folder

You can choose to clone the repository elsewhere and copy the `toothpaste.tmTheme` file from the directory.

Now, start Sublime Text and go to Preferences > Color Scheme > Toothpaste
